public/app/plugins/datasource/alertmanager/plugin.json

```diff
@@ -4,42 +4,77 @@
   "id": "alertmanager",
   "metrics": false,
   "routes": [
+    {
+      "method": "GET",
+      "path": "alertmanager/api/v2/silences",
+      "reqRole": "Viewer",
+      "reqAction": "alert.instances.external:read"
+    },
     {
       "method": "POST",
       "path": "alertmanager/api/v2/silences",
-      "reqRole": "Editor"
+      "reqRole": "Editor",
+      "reqAction": "alert.instances.external:write"
+    },
+    {
+      "method": "GET",
+      "path": "alertmanager/api/v2/silence",
+      "reqRole": "Viewer",
+      "reqAction": "alert.instances.external:read"
     },
     {
       "method": "DELETE",
       "path": "alertmanager/api/v2/silence",
-      "reqRole": "Editor"
+      "reqRole": "Editor",
+      "reqAction": "alert.instances.external:write"
     },
     {
       "method": "GET",
-      "path": "alertmanager/api/v2/silences",
-      "reqRole": "Viewer"
+      "path": "alertmanager/api/v2/alerts/groups",
+      "reqRole": "Viewer",
+      "reqAction": "alert.instances.external:read"
     },
     {
-      "method": "POST",
-      "reqRole": "Editor"
+      "method": "GET",
+      "path": "alertmanager/api/v2/alerts",
+      "reqRole": "Viewer",
+      "reqAction": "alert.instances.external:read"
     },
     {
-      "method": "PUT",
-      "reqRole": "Editor"
+      "method": "POST",
+      "path": "alertmanager/api/v2/alerts",
+      "reqRole": "Editor",
+      "reqAction": "alert.instances.external:write"
     },
     {
-      "method": "DELETE",
-      "reqRole": "Editor"
+      "method": "GET",
+      "path": "alertmanager/api/v2/status",
+      "reqRole": "Viewer",
+      "reqAction": "alert.instances.external:read"
```

Comment:
let's change it to `alert.notifications.external:read` because it returns configuration. So, it's similar to `api/v1/alerts` below